### PR TITLE
outdated: accept v1/v2 aliases for freeze-file flags

### PR DESCRIPTION
**Pull request by quasicomputational**
As suggested in #5474.

---
Please include the following checklist in your PR:

* [x] Patches conform to the [coding conventions](https://github.com/haskell/cabal/#conventions).
* [x] Any changes that could be relevant to users have been recorded in the changelog.
* [x] The documentation has been updated, if necessary.
* [x] If the change is docs-only, `[ci skip]` is used to avoid triggering the build bots.

Please also shortly describe how you tested your change. Bonus points for added tests!

### CODE REVIEW

File: cabal-install/Distribution/Client/Setup.hs [hunk 1]
```diff
@@ -1166,12 +1166,12 @@ outdatedCommand = CommandUI {
...
       (\v flags -> flags { outdatedVerbosity = v })
 
-    ,option [] ["freeze-file"]
+    ,option [] ["freeze-file", "v1-freeze-file"]
      "Act on the freeze file"
      outdatedFreezeFile (\v flags -> flags { outdatedFreezeFile = v })
      trueArg
 
-    ,option [] ["new-freeze-file"]
+    ,option [] ["new-freeze-file", "v2-freeze-file"]
```
Comment (phadej) [inline cabal-install/Distribution/Client/Setup.hs]: I'd rather make freeze files state their format than rely on user to remember it right.

**Comment by 23Skidoo** [comment]
LGTM.

File: cabal-install/Distribution/Client/Setup.hs (hunk 1 above)
Comment (23Skidoo) [reply cabal-install/Distribution/Client/Setup.hs]: Freeze file format is actually the same for v1 and v2, but the file names are different.

**Comment by 23Skidoo** [comment]
Merged, thanks!

**Pull request merged**